orchagent/portsorch.cpp

```diff
@@ -3514,6 +3514,18 @@ bool PortsOrch::initPort(const PortConfig &port)
                     m_recircPortRole[alias] = role;
                 }
 
+                if ((flex_counters_orch->getQueueCountersState()) || (flex_counters_orch->getQueueWatermarkCountersState()))
+                {
+                     auto maxQueueNumber = static_cast<uint32_t>(p.m_queue_ids.size());
+                     addPortBufferQueueCounters(p, 0, maxQueueNumber-1, false);
```

Comment:
@saksarav-nokia aren't we adding them twice, once during initPort and other via createPortBufferQueueCounters in same function ?

Reply:
> > @saksarav-nokia aren't we adding them twice, once during initPort and other via createPortBufferQueueCounters in same function ?
> 
> @vmittal-msft , The createPortBufferQueueCounters is called only for tx_host_queue and not for all. Also the tx_host_queue is not set for all paltforms.

I am making minor change to address this. Let me test it and push the commit. Looks like tx_host_queue support is added only in msft-2405 and not in master.